test/powershell/Modules/Microsoft.PowerShell.Security/certificateCommon.psm1

```diff
@@ -78,7 +78,17 @@ Function New-ProtectedCertificate
     Password: "password"
     #>
 
-    $certLocation = ".\test\tools\Modules\WebListener\ServerCert.pfx"
+    $certLocation = Join-Path ([System.IO.Path]::GetTempPath()) 'protectedCert.pfx'
+
+    # Hackery to create a new 10 digit random hex string as a password
+    $rand = [Random]::new()
+    $global:protectedCertPassword = ConvertTo-SecureString -Force -AsPlainText (((1..10).ForEach{ '{0:x}' -f $rand.Next(0xf) }) -join '')
```

Comment:
Better expose it through a function, instead of a global variable. The global variable will linger around even after you remove the module from a session.